getting_started/setup_linux/prepare_build_environment.rst

```diff
@@ -0,0 +1,104 @@
+.. _prepare-build-environment:
+
+Set up a development environment to build the kernel
+####################################################
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+************
+
+
+These instructions will help you set up a development environment for the SOF branch of the Linux kernel. If you have dedicated test hardware you can use ktest to install it over ssh, otherwise you can install it locally on your device in addition to your default kernel.
+
+Prerequisites
+*************
+
+1. Device requirements
+-----------------------
+
+**Development device:** PC running Fedora 35+ or Ubuntu 20.04+.
+
+**Target device:** PC running Fedora 35+ or Ubuntu 20.04+, with secure boot disabled. If the target device is different than the development device, they must be on the same local network so you can ssh into the target.
```

Comment:
true indeed, I've done remote debug as well. ssh support with the relevant proxy setup is enough

Reply:
I'm aware that you could use a VPN (or open the port over the network), but so are people that have that setup. If someone is familiar enough that they ssh over a vpn, they'll know that's an option. However, someone unfamiliar with ssh will most likely understand "same local network" whereas "you must have a network setup that allows an ssh connection between the dev device and the target" could confuse them. It could go either way, and I'm happy to elaborate a bit more if you think that's necessary.

Reply:
suggested edit:

You must be able to ssh into the target device, which is typically on the same local network/VPN.